operator/anarchygovernor.py

```diff
@@ -295,9 +295,13 @@ def cleanup_runs(self, runtime):
             # If run has not posted a result longer ago than the interval then do not delete
             run_post_datetime = datetime.strptime(run_resource['spec']['runPostTimestamp'], '%Y-%m-%dT%H:%M:%SZ')
             if run_post_datetime + time_interval < datetime.utcnow():
-                runtime.custom_objects_api.delete_namespaced_custom_object(
-                    runtime.operator_domain, runtime.api_version, runtime.operator_namespace, 'anarchyruns', run_name
-                )
+                try:
+                    runtime.custom_objects_api.delete_namespaced_custom_object(
+                        runtime.operator_domain, runtime.api_version, runtime.operator_namespace, 'anarchyruns', run_name
+                    )
+                except:
```

Comment:
I think this is not quite right. Needs to be:

```
except kubernetes.client.rest.ApiException as e:
```

Reply:
Added. And also try block for the deletion in cleanup_actions().